doc/source/options.rst

```diff
@@ -392,6 +392,9 @@ display.width              80           Width of the display in characters.
                                         IPython qtconsole, or IDLE do not run in a
                                         terminal and hence it is not possible
                                         to correctly detect the width.
+display.html.table_schema  True         Whether to publish a Table Schema
+                                        representation for frontends that
```

Comment:
Note that this introduces a new namespace for just this (as there is an 'html' namespace, but that is not under `display`, that is just `pd.options.html`, see the `html.border` below this). 

Maybe we could also call it 'notebook' to have it different than the 'html' (although that is probably a bad name because it is not necessarily restricted to notebook?)